src/core/tests/type_prop/arithmetic_ops.hpp

```diff
@@ -176,15 +176,29 @@ TYPED_TEST_P(ArithmeticOperator, dynamic_shape_5D) {
     ASSERT_EQ(op->get_output_partial_shape(0), (PartialShape{dynamic, 4, dynamic, dynamic, 6}));
 }
 
+TYPED_TEST_P(ArithmeticOperator, dynamic_shape_intervals) {
+    auto A = std::make_shared<op::Parameter>(
+        element::f32,
+        PartialShape{Dimension(1, 3), Dimension(1, 3), Dimension(1, 3), Dimension(4, 8), -1, 1, -1, 1, 3});
+    auto B = std::make_shared<op::Parameter>(
+        element::f32,
+        PartialShape{Dimension(1, 3), Dimension(2, 7), -1, 1, Dimension(1, 3), Dimension(4, 8), -1, 1, 3});
+
+    const auto op = std::make_shared<TypeParam>(A, B);
```

Comment:
this only covers numpy broadcasting case, right?

Reply:
Oh right, the tests for other broadcast modes are needed too. Will be added.